easybuild/easyconfigs/t/ToFu/ToFu-1.4.0-intel-2018a-Python-3.6.4.eb

```diff
@@ -0,0 +1,51 @@
+easyblock = 'PythonBundle'
+
+name = 'ToFu'
+version = '1.4.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/ToFuProject/tofu'
+description = """Tomography for Fusion."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+dependencies = [
+    ('Python', '3.6.4'),
+    ('matplotlib', '2.1.2', versionsuffix),
+]
+
+exts_default_options = {'source_urls': [PYPI_LOWER_SOURCE]}
```

Comment:
use_pip must be explicitly set in PythonBundle based easyconfigs, and should normally be set to True.